src/renderer/components/dex/TradeHistory.vue

```diff
@@ -93,10 +93,22 @@ export default {
 
       flex: 1;
 
-      .cell {
-        &.time {
+      .header {
+        .cell.time {
           text-align: right;
-          white-space: nowrap;
+        }
+      }
+
+      .body {
+        .cell {
+          font-family: FreeMono !important;
+          font-size: toRem(14px) !important;
```

Comment:
Sorry for the !important tags, but they were necessary. I suppose I could've added an id to the stuff I wanted styled, but this seemed more appropriate. There were some other stuff down the CSS line that were conflicting with these settings.